packages/core/src/index.ts

```diff
@@ -157,11 +169,6 @@ class Hopr extends EventEmitter {
     this.indexer = this.connector.indexer // TODO temporary
   }
 
-  private async startedPaymentChannels(): Promise<HoprCoreEthereum> {
```

Comment:
Get rid of promisified `this.paymentChannels` / `this.ethereum` / `this.chain` etc.